package.json

```diff
@@ -31,18 +33,24 @@
     ]
   },
   "dependencies": {
-    "codemod-cli": "^2.0.0",
+    "codemod-cli": "^2.1.0",
+    "ember-codemods-telemetry-helpers": "^1.0.1",
     "ember-template-recast": "^3.2.8",
-    "winston": "^3.2.1"
+    "sync-disk-cache": "^1.3.3",
+    "walk-sync": "^2.0.2",
+    "winston": "^3.2.1",
+    "debug": "^4.1.1"
   },
   "devDependencies": {
     "coveralls": "^3.0.7",
     "eslint": "^6.6.0",
     "eslint-config-prettier": "^6.5.0",
+    "eslint-plugin-node": "^10.0.0",
```

Comment:
This doesn’t seem used in this PR (but we should totally add it in another PR 😃 ).